user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,139 @@
+#   - AppPackage: If the catalog item is a Solution Package that can be deployed on a DataMiner System.
+#   - ProtocolPackage: If the catalog item is a Protocol that can be deployed on a DataMiner System.
+#   - Visio: If the catalog item is a Visio that can be deployed on a DataMiner System.
+type: 'AutomationScript'
+
+# [Required] 
+# The id of the catalog item.
+# All versions for the same id are shown together in the catalog.
+# This id can not be changed. 
+# If the id is not filled in, the API will return one and consider this as the registration of a new item.
+# If the id is filled in, a new version will be added to the catalog item with the given version number.
+#   Must be a valid GUID.
+id: '<fill in guid here or leave empty>'
+
+# [Required] 
+# The human-friendly name of the catalog item. 
+# Can be changed at any time.
+#   Max length: 100 characters.
+#   Can not contain newlines.
+#   Can not contain leading or trailing whitespace characters.
+title: 'add-new-switch'
+
+# [Required] 
+# The version in format X.X.X
+# Once a version has been published it can no longer be changed.
+# The version should be incremented to do any changes.
+# If you want to create a package that is not final, you can provide a label at the end: X.X.X-label
+# If a catalog item has a label in the version, then the version is considered to be a pre-release version, and not an official one.
+#   Valid versions range from 0.0.0 to 2147483647.2147483647.2147483647
+#   The label can contain characters a-z0-9 with a max length of 30 characters.
+version: '1.0.0'
+
+# [Optional]
+# General information about the catalog item.
+#   Max length: 100.000 characters
+description: |
+  This script allows you to easily add a new leaf switch element.
+
+# [Optional]
+# A short description for this specific version.
+# This message is shown along side each version in the version overviews in the catalog. 
+#   Max length: 500 characters.
+version-message: 'Initial version. Does not contain anything useful yet.'
+
+# [Optional]
+# A valid URL that points to the home page.
+# Usually, this home page is also the location of the source code.
+#   A valid URL
+#   Max length: 2048 characters
+homepage: 'https://github.com/randomrepo'
+
+# [Optional]
+# People that are responsible for this catalog item. Might be developers but is not required.
+# Format: 'name <email> (url)'
+#   The name is required, max 256 characters.
+#   The email and url are optional, and should be in valid email/URL formats.
+owners:
+  - name: 'David Smith'
+  - name: 'Someone'
+    email: 'someone@skyline.be'
+  - name: 'John Smith'
+    email: 'john@skyline.be'
+    url: 'https://github.com/JohnSmith'
+
+# [Optional]
+# Tags that allow you to categorize your catalog items.
+#   Max length: 50 characters.
+#   Can not contain newlines.
+#   Can not contain leading or trailing whitespace characters.
+tags:
```

Comment:
I'm a little bit worried about the amount of tags will potentionally be introduced. How will you cope with this on the Catalog UI?

Reply:
We can opt to only show the first x amount of tags.

Reply:
We can limit this in the first version(s), however, this means we'll need a clear list of which tags are available.
ATM we're not limiting what the users can provide for tag(s)